resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -241,6 +250,57 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class Scrollbars {
+
+    [DscProperty(Key)] [Status] $AlwaysShow = [Status]::KeepCurrentValue
+
+
+    hidden [string] $DynamicScrollbarsProperty = 'DynamicScrollbars'
+
+    [AlwaysShowScrollbars] Get() {
+        $currentState = [AlwaysShowScrollbars]::new()
+        
+		if (-not(DoesRegistryKeyPropertyExist -Path $global:ControlPanelAccessibilityRegistryPath -Name $this.DynamicScrollbarsProperty)) {
+            $dynamicScrollbarsValue = [BinarySettingState]::Disabled                
+        } else {
+			$dynamicScrollbarsValue = (Get-ItemProperty -Path $global:ControlPanelAccessibilityRegistryPath -Name $this.DynamicScrollbarsProperty).DynamicScrollbars
+			$currentState.ShowScrollBars = switch ($dynamicScrollbarsValue) {
+				1 { [BinarySettingState]::Disabled }               
+				0 { [BinarySettingState]::Enabled }               
+				default { [BinarySettingState]::KeepCurrentValue }
+			}

```

Comment:
```suggestion
			$currentState.Show = ($dynamicScrollbarsValue -eq 0) ? $true : $false

```

Reply:
$currentState.DynamicScrollbarState = [BinaryState].getEnumName((Get-ItemProperty -Path $global:ControlPanelAccessibilityRegistryPath -Name $this.DynamicScrollbarsProperty).DynamicScrollbars)